maintenance_location/views/maintenance_request.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 Creu Blanca
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record model="ir.ui.view" id="hr_equipment_request_view_form">
+        <field name="name">maintenance.request.form (in maintenance_location)</field>
+        <field name="model">maintenance.request</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_request_view_form" />
+        <field name="arch" type="xml">
+            <field name="company_id" position="after">
```

Comment:
I think that you need to use a xpath because the first "company_id" is invisible and is before the header:

https://github.com/odoo/odoo/blob/16.0/addons/maintenance/views/maintenance_views.xml#L58

Reply:
[resolved]

Reply:
Thanks!